### DISCUSSION TITLE
뉴비답게 Hell World를 로그에 찍어보려고 하는데요

### DISCUSSION THREAD

**Discussion by simeddk** [simeddk/Tore04_BasicSyntaxCPP#5] [Q&A] [answered]
아예 컴파일이 안되네요....

```c++
FString str = "Hell World";
UE_LOG(LogTemp, Display, TEXT("%s"), str);
```

왜 하필 저에게 이런 일이 발생하는걸까요..

**Comment by HyeonLang** [comment, marked as answer]
김재우
str -> *str : FString -> const char *

**Comment by ehddnjs1200** [comment]
정동원

UE_LOG를 사용할 때는 FString을 직접 전달할 수 없습니다. 대신, FString의 * 연산자를 사용하여 FString을 TCHAR*로 변환하면
UE_LOG에서 FString을 전달이 가능합니다
